spec/contracts/custom_fields/hierarchy/insert_item_contract_spec.rb

```diff
@@ -67,15 +67,27 @@
       it "is invalid" do
         result = subject.call(params)
         expect(result).to be_failure
-        expect(result.errors.to_h).to include(label: ["must be unique within the same hierarchy level."])
+        expect(result.errors.to_h).to include(label: [I18n.t("dry_validation.errors.rules.label.not_unique")])
       end
 
-      context "if locale is set to 'de'" do
+      context "if another locale is set" do
+        let(:mordor) { "agh burzum-ishi krimpatul" }
+
+        before do
+          I18n.config.enforce_available_locales = false
```

Comment:
This change will most likely (didn't check) spill over to the specs running after this one. So it should be reverted in an `after` block.

Reply:
sure, thanks for being very vigilant